frontend/src/components/logout/Logout.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const Logout = () => (
```

Comment:
Looks much similar to the Login component. Also the text "Log out successful" looks duplicated.